examples/text-generation/mbxp_evaluation/evaluation_setup/ubuntu.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/bash
```

Comment:
This is uncommon to create bash scripts in examples. @regisss can you please help to decide what to do with these?
We can put them directly in makefile or just put the instructions in README?